speech/grpc/src/main/proto/google/longrunning/operations.proto

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2015, Google Inc.
```

Comment:
This has been resolved separately. @lesv please confirm if still any reservations.